ext/opentelemetry-ext-wsgi/README.rst

```diff
@@ -1,9 +1,20 @@
 OpenTelemetry WSGI Middleware
 =============================
 
+.. |pypi| image:: https://badge.fury.io/py/opentelemetry-opentracing-wsgi.svg
```

Comment:
This one doesn't render. Missing a `[pypi]` above?

Reply:
You're right, thanks!